nav.css

```diff
@@ -0,0 +1,69 @@
+
+/* navbar*/
+.navbar-top {
+	border: 2px solid black;
+}
+
+.navbar-top {
+	display: flex;
+	justify-content: space-between;
+	height: 60px;
+	line-height: 60px;
+	background-color:black;
+}
+#top1 a {
+	font-size: 25px;
+	margin-left: 50px;
+}
+#top11 {
+	color: #686;
+}
+#top2 a:first-child {
+	font-size: 18px;
+	margin-right: 30px;
+	background-color: #686;
+	padding: 10px 20px;
+	border-radius: 20px;
+	transition: background-color 2s linear;
+}
+#top2::first-letter {
+	text-transform: capitalize;
+}
+#top2 a:first-child:hover {
+	background-color: green;
+}
+#top2 a:last-child {
+	margin-right: 50px;
+	color: #777777;
+	font-size: 14px;
+	transition: color 2s linear;
+}
+#top2 a:last-child:hover {
+	color: white;
+}
+/* logo-botton*/
+
+.logo-botton {
+	height: 60px;
+	line-height: 60px;
+	display: flex;
+	justify-content: space-between;
+	margin-left: 50px;
```

Comment:
Pytanie czy oby na pewno chcemy przesunąć w prawo całe menu? Jakby to menu nagle zmieniło kolor na czerwony - jakby to wyglądało? Oprócz tego, przesunięcie całego elementu wpływa na widok mobilny (negatywnie).